doc/model.md

```diff
@@ -1,516 +1,247 @@
+into this repository.
 
-A list of valid locales can be viewed in the Publishing API [here](https://github.com/alphagov/publishing-api/blob/9caeccc856ad372e332f56d64d0a96ab5df76b27/config/application.rb#L32-L37).
-This field is not required and if a `locale` is not provided, it will be set to
-*en* automatically.
+![Diagram of the object model](model/object-model.png)
 
-### `need_ids`
+# Content
 
-Example: *["1234", "1235"]*
+## Document
 
-Required: No
+A document represents all iterations of a piece of content in a particular
+locale. It is associated with multiple editions that represent distinct
+versions of a piece of content.
 
-The `need_ids` are the identifiers of [user needs](https://www.gov.uk/design-principles)
-that are entered through the [Maslow application](https://github.com/alphagov/maslow).
-They are passed through to the content store, untouched by the pipeline. The
-front-end applications can then use the `need_ids` to present pages to users
-that show how effectively users' needs are being met. [Here is an example](https://www.gov.uk/info/overseas-passports).
+The concerns of a document are which iterations are represented on draft and
+live content stores; and the [lock version][optimistic-locking] for the content.
 
-### `phase`
+A document stores the [`content_id`](#content_id), locale and lock version for
+content. It is designed to be a simple model so that it can be used for
+database level locking of concurrent requests.
 
-Examples: *alpha, beta, live*
+## Edition
 
-Required: No
+An edition is a particular iteration of a piece of content. It stores most
+of the data that is used to represent content in the content store and is
+associated with a document. There are [uniqueness constraints](#uniqueness)
+to ensure there are not conflicting Editions. Previously an Edition was named
+ContentItem.
 
-The `phase` is an optional field that may be used to indicate the ‘Service
-Design Phase’ of the content item. The phase must be one of either *alpha*,
-*beta* or *live*. If the `phase` is not specified, the Publishing API will
-default it to *live*.
+Most of the fields stored on an edition are defined as part of the
+[/put-content/:content_id][put-content-api] API.
 
-Content items will be published to the content store regardless of their
-`phase`.  If a content item has a `phase` of either *alpha* or *beta*, a visual
-component will be added to the front-end to show this (assuming the front-end
-application for that format supports this).
+Key fields that are set internally by the Publishing API are:
 
-There is more information on what each of the phases mean [here](https://www.gov.uk/service-manual/phases).
+- `state` - where an edition is in it's [publishing workflow](#workflow), can
+  be "draft", "published", "unpublished" or "superseded".
+- `content_store` - indicates whether an edition is on the draft or live
+  content store, or empty to indicate that it is not on a content store.
```

Comment:
Could probably do with some more detail here. If something has a value of "live", it may be in both content stores.